api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -75,6 +78,27 @@ public void validate(Object target, Errors errors) {
 			errors.rejectValue("stopDatetime", "Visit.error.endDateBeforeStartDate");
 		}
 		
+		try 
+		{
+			Date birthDateTime = visit.getPatient().getBirthdate();
+			Date startDateTime = visit.getStartDatetime();
+			if(startDateTime.compareTo(birthDateTime) < 0) {
```

Comment:
You will also need to run mvn clean install before committing such that it auto formats the source.